src/components/Settings.vue

```diff
@@ -4,35 +4,35 @@
     <v-card-text>
       <OpenImage @change="onImageOpen"
         @delete="ImageDelete"
-        :ImageCheck = "ImageCheck"/>
-      <ImageProcessor :imageUrl = "imageUrl" @imageSampled="onImageSampled"/>
+        :imageCheck="imageCheck"/>
+      <ImageProcessor
+        :image-width="imageWidth"
+        :image-height="imageHeight"
+        :image-url="imageUrl"
+        @imageSampled="onImageSampled"/>
       <v-layout row wrap>
           <v-flex xs3 style="margin: 24px;">
               <v-text-field
               label="Width"
               id="widthSetting"
-              placeholder="32"
+              placeholder="64"
               type="number"
-              value=32
-              @change="onDimChange"
+              v-model="imageWidth"
+              @change="validateWidth"
               min=0
               max=5000
```

Comment:
These should reflect `maxWidth` and `minWidth`

Reply:
Changed.

Reply:
Can we do `:min="minWidth"`, etc. so we don't repeat these values?

Reply:
Doing some testing... it looks like v-model in v-text-field doesn't respect the `min` or `max` values at all. I think this is why I had to implement manual validation...

Reply:
[Vuetify's official solution (as I found it) seems to be to attach to `@Focus`](https://github.com/vuetifyjs/vuetify/issues/576#issuecomment-440073654).

Reply:
Oh good point, I think the only benefit it really provides for us is it prevents the up/down steppers from going out of the bounds. The browser's built-in validation only applies on submit.

Reply:
What's the verdict? The basic functionality is there, but bulletproof input validation might take more work to implement.